Code Clean up

vish288 opened the pull request:
Indentation consistency
Removal of Lint issues.
Es6 literal usage.

mfeniseycopes committed:
Refactor index.js

mfeniseycopes committed:
Fix noLogger ordering

mfeniseycopes committed:
Have option checkers return boolean

mfeniseycopes committed:
Add more specs/reorganize

mfeniseycopes committed:
noLogger -> hasLogger

Commit:
Code Clean up
Indentation consistency
Removal of Lint issues.
Es6 literal usage.

Commit:
Merge branch 'refactor'

# Conflicts:
#	src/index.js

vichu1988 commented:
updated the code to include the commit from https://github.com/mfeniseycopes/redux-logger/tree/refactor as well. #223

vichu1988 commented:
@evgenyrodionov : The issue seems to be with codecov.io not having a comparison. 
Apart from that could you comment on the code pls?

Commit:
Fixes:

1. Update dependencies to the latest version.
2. Upgrade babel
3. Upgrade eslint
4. Upgrade Rollup
5. Cleanup rollup config.
6. Introduce Terser instead of Uglify

Commit:
Added in shrinkwrap and lint issues clear.

Commit:
update dependencies and cleanup

Commit:
lint cleanup diff
